### ISSUE TITLE
Menu: Several triggers not supported

### ISSUE THREAD

**Issue by platon-rov** [SAP/fundamental-ngx#5998]
#### Is this a bug, enhancement, or feature request?

Enhancement

#### Briefly describe your proposal.

It may be useful to be able to have several menu triggers and only one menu component. As for now, we need to have trigger for every single menu, in case where we have a lot of triggers present in the DOM we will have a lot of menus.

[StackBlitz](https://stackblitz.com/edit/angular-aaccaf) example from the Angular Material.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

0.31.0-rc.101

#### If this is a bug, please provide steps for reproducing it.

.

#### Please provide relevant source code if applicable.

https://stackblitz.com/edit/angular-aaccaf

#### Is there anything else we should know?

.

**Comment by droshev** [comment]
@platon-rov Can you summarize what exactly we have to implement?
